### ISSUE TITLE
react-dom.production.min.js:187 TypeError: Cannot read property 'root' of null

### ISSUE THREAD

**Issue by chrisgorgo** [OpenNeuroOrg/openneuro#179]
When accessing https://openneuro.org/datasets/ds001131/versions/00001?app=MRIQC&version=46&job=71c23039-7502-4f8e-8733-495046c9fbf5
(while not logged in)
```
react-dom.production.min.js:187 TypeError: Cannot read property 'root' of null
    at t.value (index.js:38)
    at h (react-dom.production.min.js:147)
    at beginWork (react-dom.production.min.js:150)
    at c (react-dom.production.min.js:182)
    at g (react-dom.production.min.js:183)
    at h (react-dom.production.min.js:184)
    at d (react-dom.production.min.js:188)
    at a (react-dom.production.min.js:187)
    at Object.enqueueSetState (react-dom.production.min.js:140)
    at t.setState (react.production.min.js:12)
```

**Comment by chrisgorgo** [comment]
Goes away if logged in.

**Comment by nellh** [comment]
This was just fixed in #178 and is rolling out now.
